### PR TITLE
Add documentation for HS01B-1

### PR DESCRIPTION
**Pull request by staxDB**
- Added documentation for HS01B-1
- Added yaml-example for HS01B-1 (`flexispot_e5b.yaml`)
- Update `flexispot_ek5.yaml` to use correct M-command

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,34 +56,33 @@ If your control panel is missing, feel free to [create an issue](https://github.
...
+| 5        | TX         | Red                  | Blue                         |
+| 4        | GND1       | Green                | White-Blue                   |
+| 3        | +5V (VDD)  | Black                | Green                        |
+| 2        | 29V+       | Blue                 | White-Brown                  |
+| 1        | 29V-       | Yellow               | Brown                        |
 
 #### [HS01B-1](https://www.loctekmotion.com/shop/control-panel/hs05a-1/)
 
 - **Desk model**: Flexispot E5B
 - **Tested with control box**: unknown
```
Comment (iMicknl) [inline README.md:73]: @staxDB do you perhaps know which control box you have?

File: packages/esphome/flexispot_ek5.yaml [hunk 2]
```diff
@@ -119,7 +119,7 @@ switch:
     name: "M"
     id: switch_m
     icon: mdi:alpha-m-circle
-    data: [0x9b, 0x06, 0x02, 0x20, 0x00, 0xac, 0xb8, 0x9d]
+    data: [0x9b, 0x06, 0x02, 0x00, 0xac, 0xac, 0xb8, 0x9d]
```
Comment (iMicknl) [inline packages/esphome/flexispot_ek5.yaml:122]: I would need to double check this in my example. The M function works via ESPHome, thus not sure if my control box has slightly different commands.

**Review by iMicknl — COMMENTED** [review]
Thanks @staxDB! :)

File: README.md (hunk 1 above)
Comment (staxDB) [reply README.md:73]: I have the model CB38M2A-1

File: packages/esphome/flexispot_ek5.yaml (hunk 2 above)
Comment (staxDB) [reply packages/esphome/flexispot_ek5.yaml:122]: I have followed this table:
https://github.com/iMicknl/LoctekMotion_IoT#command-list

How should the M-Key actually work? Do I first click on a preset, then set the height and then click on the M-Key?

Comment (iMicknl) [reply packages/esphome/flexispot_ek5.yaml:122]: It should work the same as with the physical controller. You set a height, than press the "M" button and than choose the preset to overwrite. 

I will check if my control panel works with the other M button, or if this wasn't documented properly for my control panel.

Comment (staxDB) [reply packages/esphome/flexispot_ek5.yaml:122]: If I look [here](https://github.com/staxDB/standing-desk-interceptor#button-presses), the original signal seems to have been correct. The main README needs to be updated here: https://github.com/iMicknl/LoctekMotion_IoT#command-list

Comment (staxDB) [reply packages/esphome/flexispot_ek5.yaml]: I have just tested it: Your original signal was correct, now the M-Key also works for me.

**Review by iMicknl — APPROVED** [review]

**Comment by iMicknl** [comment]
Thanks @staxDB, great to hear that it works for you :). Do you power your device via USB or the 5V output?

**Comment by staxDB** [comment]
Currently I'm using the 5V output. Here are a few pictures of my wiring:
image
image

**Comment by iMicknl** [comment]
Cool! I didn't try it yet, but will give it a try :).

**Pull request merged**